functions/firebase_analytics/composer.json

```
@@ -0,0 +1,11 @@
+{
+    "require": {
+        "google/cloud-functions-framework": "^0.7.1"
```

nit: update to `^0.7.2`
```suggestion
        "google/cloud-functions-framework": "^0.7.2"
```